### PR TITLE
Dreamworks: Add DreamworksF722 board config support

### PR DESCRIPTION
**Pull request by Dreamworks-fpv**
Hello everyone, we are a start-up hardware team. We have recently released a new flight controller and request to be added to the repo so that users can quickly find and download it.

### CODE REVIEW

File: configs/default/DREA-DreamworksF722.config [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+#define USE_GYRO_SPI_ICM20689
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+
+board_name DreamworksF722
+manufacturer_id Drea
```
Comment (nerdCopter) [inline configs/default/DREA-DreamworksF722.config:15]: All caps.
needs to be officially accepted to https://github.com/betaflight/unified-targets/blob/master/Manufacturers.md as well.

Comment (haslinghuis) [reply configs/default/DREA-DreamworksF722.config:15]: @Dreamworks-fpv please provide an URL where this hardware is available.

Comment (Dreamworks-fpv) [reply configs/default/DREA-DreamworksF722.config:15]: #1177

Comment (haslinghuis) [inline configs/default/DREA-DreamworksF722.config:15]:
```suggestion
board_name DREAMWORKSF722
manufacturer_id DREA
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/DREA-DreamworksF722.config (hunk 1 above)
Comment (Dreamworks-fpv) [reply configs/default/DREA-DreamworksF722.config]: Fixed and rebased it. Thanks your suggestion~!

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by Dreamworks-fpv** [comment]
Thanks all.

**Pull request merged**